andela/wg-justice-league
# #154240263  Add support for macrocycle, mesocycle and microcycle

#### What does this PR do?
- This PR enables a user to choose a schedule plan  for a certain period they intend to work out for.
#### How should this be manually tested?
- Run Wger application
- Navigate to  `Workout Schedules` via the `Workout` tab .
- Click on `add schedule` and check `Schedule Plan`.
- Select your plan either `macrocycle`or `mesocycle` or `microcycle`
#### What are the relevant pivotal tracker stories?
- #154240263
#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/30471763/35676382-a51cf208-075c-11e8-8803-b092fdc77767.png)
![image](https://user-images.githubusercontent.com/30471763/35677326-30447312-0760-11e8-84e6-d24b7f8c93ef.png)
